scripts/cold_boot_opto_original.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+echo "CTP7 Virtex-7 cold boot in progress..."
+
+echo "Configuring reference clocks..."
+
+# First initialize ref clocks before loading the V7 firmware (160 MHz refclk)
+clockinit 320_160 320_160 B1 A1 A1 B1
+
+sleep 1
+
+false
+RETVAL=$?
+
+while [ $RETVAL -ne 0 ]
+do
+    v7load tamu_ctp7_v7_original.bit
+    RETVAL=$?
+done
+
+# Disable Opto TX Lasers
+/bin/txpower enable
+
+# Configure GTHs in loopback mode
+sh gth_config_opto.sh
+
+# GTH channel reset procedure
+sh gth_reset.sh
+
+#Print gth status register
+sh gth_status.sh
+
```

Comment:
Should be updated similar to how I updated cold_boot.sh so that it is aware of where the script lives, as it expects to be run from a specific location